### PR TITLE
Improve & simplify readme

### LINKED ISSUE #593: README:  Refactor, better structure
Let's clean main README, and structure it better

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #593 

Changes proposed in this PR:

- Improve & simplify the main readme (including only necessary information)
- Moving everything else to other files in the /docs directory

Preview [here](https://github.com/oceanprotocol/ocean-node/tree/improve-readme?tab=readme-ov-file#ocean-nodes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,446 +1,96 @@
...
 
-```bash
-npm i
-```
+1. **Build the Docker image:**
+   `docker build -t ocean-node:mybuild .`
 
-### 3. Build
+2. **Run the Docker container:**
+   `docker run -e PRIVATE_KEY=your_private_key_here ocean-node:mybuild`
```
Comment (jamiehewitt15) [inline README.md:17]: I'll update this to include the docker quick start script that @md00ux is working on when it is ready.

File: README.md [hunk 2]
```diff
@@ -1,446 +1,96 @@
...
-A full list of all environmental variables is available in [env.md](./env.md)
+`export PRIVATE_KEY="your_private_key_here"`
 
-For now, we have three private keys defined (NODE1_PRIVATE_KEY, NODE2_PRIVATE_KEY,NODE3_PRIVATE_KEY). They are using the 7th, 8th and 9th accounts of barge:
+The `PRIVATE_KEY` is the only mandatory environmental variable. Additional configurations can be set as needed.
 
-- (7) 0x1d751ded5a32226054cd2e71261039b65afb9ee1c746d055dd699b1150a5befc
-- (8) 0xfd5c1ccea015b6d663618850824154a3b3fb2882c46cefb05b9a93fea8c3d215
-- (9) 0x1263dc73bef43a9da06149c7e598f52025bf4027f1d6c13896b71e81bb9233fb
+For all available configurations, refer to the [Environment Variables](docs/envs.md) documentation.
```
Comment (bogdanfazakas) [inline README.md:67]:
```suggestion
For all available configurations, refer to the [Environment Variables](docs/env.md) documentation.
```

File: docs/testing.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+```bash
+npm run test:request:rate
+```
+
+At the end of the test suite, you can check the generated HTML report `html-report.html` for more insigths.
+Additionally, while the tests are running you can open
+a browser page at `http://127.0.0.1:5665/` and see a live report
+
+For a more detailed view of all the options available and the type of requests executed check the script: [./src/test/performance/util.js](./src/test/performance/util.js)
```
Comment (bogdanfazakas) [inline docs/testing.md:75]:
```suggestion
For a more detailed view of all the options available and the type of requests executed check the script: [../src/test/performance/util.js](../src/test/performance/util.js)
```

File: README.md [hunk 4]
```diff
@@ -1,446 +1,96 @@
...
-wWe6BFQXZWg6zE9X7ExynvXEe9BRTR5Wn3udNs7JpUDx
-```
+- [Ocean Nodes Architecture](docs/Arhitecture.md)
+- [API Endpoints](docs/API.md)
+- [Environmental Variables](docs/env.md)
+- [Testing Guide](docs/testing.md)
+- [Network Configuration](docs/networking.md)
+- [Logging & accessing logs](docs/networking.md)
+- [Dashboard: Local development](dashboard/README.md)
+- [Compute to Data V2](docs/C2DV2.md)
```
Comment (bogdanfazakas) [inline README.md:96]: should we reference this in the main readme atm or we should wait for it to be implemented ?

Comment (jamiehewitt15) [reply README.md:96]: Yeah good point, I'll remove it for now

**Review by bogdanfazakas — APPROVED** [review]
lgtm, there are some README conflicts that need to be addressed most likely coming from output stage pr

File: README.md [hunk 5]
```diff
@@ -1,447 +1,123 @@
...
-export RPCS="{ \"1\":{ \"rpc\":\"https://rpc.eth.gateway.fm\", \"chainId\": 1, \"network\": \"mainet\", \"chunkSize\": 100 }, \"137\": { \"rpc\": \"https://polygon.meowrpc.com\", \"chainId\": 137, \"network\": \"polygon\", \"chunkSize\": 100 }, \"80001\": { \"rpc\": \"https://rpc-mumbai.maticvigil.com\", \"chainId\": 80001, \"network\": \"polygon-mumbai\", \"chunkSize\": 100 }}"
+pm2 list # View running processes
+pm2 logs ocean-node # View logs
+pm2 restart ocean-node # Restart the node
+pm2 stop ocean-node # Stop the node
+pm2 delete ocean-node # Delete the process
 ```
 
-Network interfaces supported by the node ('http' and/or 'p2p'). By default, if not specified otherwise, both are supported. Case insensitive.
+## Running Ocean Nodes Locally
```
Comment (mihaisc) [inline README.md:47]: i would suggest removing "Locally" it's the same wherever you are deploying it

Comment (jamiehewitt15) [reply README.md:47]: Yeah, I was looking for a title that differentiates this from the other ways of running the nodes. I'll change it to `running the node with npm`

File: README.md [hunk 6]
```diff
@@ -1,447 +1,123 @@
...
-For purgatory checks, please export the following env variables;
-
-```bash
-export ASSET_PURGATORY_URL=\"https://raw.githubusercontent.com/oceanprotocol/list-purgatory/main/list-assets.json\"
-export ACCOUNT_PURGATORY_URL=\"https://raw.githubusercontent.com/oceanprotocol/list-purgatory/main/list-accounts.json\"
-```
+### 3. Start Required Services
 
-For configuring the ocean node fees, please export the following environment variables;
+In a separate terminal, clone and start the necessary services using Barge:
```
Comment (MBadea17) [inline README.md:67]: For clarity, I would mention that the deployment of the barge is necessary only if they want to run the node connected to the local chain.

Comment (bogdanfazakas) [reply README.md:67]: if you wnat to run it with indexer you will also need barge for the database

**Comment by mihaisc** [comment]
good to go?

**Pull request merged**